packages/twenty-server/src/engine/core-modules/invitation/invitation.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+import { IDField } from '@ptc-org/nestjs-query-graphql';
+import {
+  CreateDateColumn,
+  Entity,
+  OneToOne,
+  PrimaryGeneratedColumn,
+  Relation,
+  UpdateDateColumn,
+} from 'typeorm';
+
+import { UUIDScalarType } from 'src/engine/api/graphql/workspace-schema-builder/graphql-types/scalars';
+import { AppToken } from 'src/engine/core-modules/app-token/app-token.entity';
+
+@Entity({ name: 'invitation', schema: 'core' })
+@ObjectType('Invitation')
+export class Invitation {
```

Comment:
I would challenge if this is really necessary, we try to be cautious not to create tables unless they are stricly required